### PR TITLE
Story 1452: Add connect

### PR DESCRIPTION
**Pull request by benediktpascher**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,8 +47,8 @@ <h1>Your profile:</h1>
   }
 
   function showLoginButton() {
-    const CONNECT_URL = 'https://uppers-alpha.auth.eu-central-1.amazoncognito.com'; // Will be provided by Uplyft
-    const CONNECT_CLIENT_ID = '7d3gktssvvba6emj7ptqjhccg4'; // Will be provided by Uplyft
+    const CONNECT_URL = 'https://connect.uppers.alpha.uplyft.com'; // Will be provided by Uplyft
+    const CONNECT_CLIENT_ID = '1novo4atmn4e38k1bked20rtj'; // Will be provided by Uplyft
     const RESPONSE_TYPE = 'token';
```
Comment (mathiasmoeller) [inline index.html:52]: We don't offer the response type prop I think?

**Review by mathiasmoeller — COMMENTED** [review]
Does the documentation have to be updated as well? (README)

File: index.html (hunk 1 above)
Comment (benediktpascher) [reply index.html:52]: Right

**Review by mathiasmoeller — APPROVED** [review]
Did you test if it is working with the new config?

**Comment by benediktpascher** [comment]
Yes, I've tested it temporarily, it works.

**Pull request merged**